packages/theme/style.scss

```diff
@@ -0,0 +1,25 @@
+:root {
+	--wp-theme-color-neutral-bg-surface: var(--wp-theme-color-neutral-1);
```

Comment:
Looking at the naming convention, what would be other values that could be used instead of "neutral"? Would "primary" be such an alternative? If so, why aren't there any `--wp-theme-color-primary` variables in this file?